### PR TITLE
PMM-6380 Release Notes 2.9.1

### PR DESCRIPTION
**Pull request by PaulJacobs-percona**

### CODE REVIEW

**Comment by PaulJacobs-percona** [comment]
Preview:
https://new.percona.com/doc/percona-monitoring-and-management/2.x/release-notes/2.9.1.html

File: source/release-notes/2.9.1.rst [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+:Installation: `Installing Percona Monitoring and Management <https://www.percona.com/doc/percona-monitoring-and-management/2.x/install/index-server.html>`_
+
+*PMM* (`Percona Monitoring and Management <https://www.percona.com/doc/percona-monitoring-and-management/index.html>`_)
+is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL
+performance.
+
+Improvements
+================================================================================
+
+* :jirabug:`PMM-6230`: Home Dashboard: Customizations retained across updates
```
Comment (rnovikovP) [inline source/release-notes/2.9.1.rst:17]: it's not about the Home dashboard. you can't edit it. its  about :  You  created a NEW  dashboard for you, set it  as the HOME page and it's now  preserved as HONE dashboard after update

File: source/release-notes/2.9.1.rst [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+* :jirabug:`PMM-6300`: Query Analytics Dashboard: Column sorting arrows made easier to use (Thanks to user debug for reporting this issue)
+* :jirabug:`PMM-6208`: Security Threat Tool: Silence viewed but unactioned alerts
```
Comment (rnovikovP) [inline source/release-notes/2.9.1.rst:19]: Alert will appear in 24h is the source of the problem will not be fixed. so this is temporary  hide the alert from the panel

File: source/release-notes/2.9.1.rst [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+* :jirabug:`PMM-6211`: Query Analytics Dashboard: Loading activity spinner for Example, Explain and Tables tabs
+* :jirabug:`PMM-6162`: Consistent sort order in dashboard drop-down filter lists
+* :jirabug:`PMM-5132`: Better message when filter search returns nothing
+
+
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PMM-5783`: Bulk failure of SHOW ALL SLAVES STATUS scraping on PS/MySQL distributions triggers errors
```
Comment (rnovikovP) [inline source/release-notes/2.9.1.rst:34]: .. was causing a lot of errors in logs and alerting in different systems

Comment (PaulJacobs-percona) [reply source/release-notes/2.9.1.rst:34]: That's already said. You want to add 'which fills up logs and affects other systems'?

Comment (rnovikovP) [reply source/release-notes/2.9.1.rst:34]: sorry, lost  the next  line . no, it's OK

File: docs/release-notes/2.9.1.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+* [PMM-6300](https://jira.percona.com/browse/PMM-6300): Query Analytics Dashboard: Column sorting arrows made easier to use (Thanks to user debug for reporting this issue)
+* [PMM-6208](https://jira.percona.com/browse/PMM-6208): Security Threat Tool: Temporarily silence viewed but unactioned alerts
+* [PMM-6315](https://jira.percona.com/browse/PMM-6315): Query Analytics Dashboard: Improved metrics names and descriptions
+* [PMM-6274](https://jira.percona.com/browse/PMM-6274): MySQL User Details Dashboard: View selected user's queries in Query Analytics Dashboard
+* [PMM-6266](https://jira.percona.com/browse/PMM-6266): Query Analytics Dashboard: Pagination device menu lists 25, 50 or 100 items per page
+* [PMM-6262](https://jira.percona.com/browse/PMM-6262): PostgreSQL Instance Summary Dashboard: Descriptions for all 'Temp Files' views
+* [PMM-6253](https://jira.percona.com/browse/PMM-6253): Query Analytics Dashboard: Improved SQL formatting in Examples panel
+* [PMM-6211](https://jira.percona.com/browse/PMM-6211): Query Analytics Dashboard: Loading activity spinner for Example, Explain and Tables tabs
```
Comment (shoffman-percona) [inline docs/release-notes/2.9.1.md:35]: not sure I understand what this is...was it added, improved, changed, removed?

Comment (PaulJacobs-percona) [reply docs/release-notes/2.9.1.md:35]: It's new. I'll make that clearer.

File: docs/release-notes/2.9.1.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+<table class="docutils field-list" frame="void" rules="none">
+  <colgroup>
+    <col class="field-name">
+    <col class="field-body">
+  </colgroup>
+  <tbody valign="top">
+    <tr class="field-odd field">
+      <th class="field-name">Date:</th>
+      <td class="field-body">August 3, 2020</td>
```
Comment (alexanderblinkov-percona) [inline docs/release-notes/2.9.1.md:11]: The date has to be changed to August 4, 2020

**Pull request merged**